app/views/pets/index.html.erb

```diff
@@ -1,13 +1,16 @@
 <h1>All Pets</h1>
-<% @pets.each do |pet| %>
-  <ul class="pet_profile">
-    <img src="<%= pet.image_path %>" alt="<%= pet.image_path %>" class="pet_index_image">
-    <li>Name: <%= link_to "#{pet.name}", "/pets/#{pet.id}"%></li>
-    <li>Approximate age: <%= pet.approximate_age %></li>
-    <li>Sex: <%= pet.sex %></li>
-    <li>Shelter: <%= link_to "#{pet.shelter.name}", "/shelters/#{pet.shelter.id}" %></li>
-    <li><%= link_to "Update Pet", "/pets/#{pet.id}/edit", id: "update-pet-#{pet.id}"%></li>
-    <li><%= link_to "Delete Pet", "/pets/#{pet.id}", id: "delete-pet-#{pet.id}", method: :delete %></li>
-  </ul>
-  <br>
-<% end %>
+
+<div class="pets_list">
+  <% @pets.each do |pet| %>
+    <ul class="pet_profile">
```

Comment:
I made this class so that I could test which pet came first to make sure the pets who aren't adoptable come last.